keymap.cson

```diff
@@ -26,6 +26,8 @@
 
 # window/pane navigation
 'body, atom-text-editor':
+  'shift-cmd-y': 'pane:show-next-item'
+  'shift-cmd-h': 'pane:show-previous-item'
```

Comment:
yeah, i'm debating whether or not i want to try and use those. those are in the worst place on the kinesis...